js/main.js

```diff
@@ -0,0 +1,93 @@
+var categoriesRequest = new XMLHttpRequest();
+var questionsRequest = new XMLHttpRequest();
+var $categorySelect = document.querySelector('#category-select');
+var $triviaForm = document.querySelector('#trivia-form');
+var $questionContainer = document.querySelector('.question-container');
+var $setUpContainer = document.querySelector('.container');
+
+var category = '';
```

Comment:
Why is this out here? You should just declare this variable in the loop where you are using it.